std_msgs/Header header
int32[] pedestrian_id
nav_msgs/Path[] path